starter-repos/anthos-config-management/namespaces/app-environment/namespace.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Google LLC
+#
```

Comment:
What is the purpose of the `app-environment` dir?

Reply:
This namespace is for the Environment CR. It is a separate namespace.

Reply:
I see. Can you add this into this yaml file as a comment?